pkg/cmd/experimental/registry/registry.go

```diff
@@ -140,6 +143,18 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 		label = valid
 	}
 
+	nodeSelector := map[string]string{}
+	if len(cfg.Selector) > 0 {
+		valid, remove, err := app.LabelsFromSpec(strings.Split(cfg.Selector, ","))
+		if err != nil {
+			glog.Fatal(err)
```

Comment:
I’ll make the change here and create an issue to update the router and any other commands.

> On May 27, 2015, at 1:21 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/experimental/registry/registry.go <https://github.com/openshift/origin/pull/2502#discussion_r31157800>:
> 
> > @@ -140,6 +143,18 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
> >  		label = valid
> >  	}
> >
> > +	nodeSelector := map[string]string{}
> > +	if len(cfg.Selector) > 0 {
> > +		valid, remove, err := app.LabelsFromSpec(strings.Split(cfg.Selector, ","))
> > +		if err != nil {
> > +			glog.Fatal(err)
> should always bubble errors to a checkErr at the top level of the command
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/openshift/origin/pull/2502/files#r31157800>.
>